### PR TITLE
dev-python/dask: version bump to 1.1.3 EAPI bump 7

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.62, Repoman-2.3.12
Signed-off-by: Horea Christian <horea.christ@yandex.com>

### CODE REVIEW

File: dev-python/dask/dask-1.1.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6} )
```
Comment (arthurzam) [inline dev-python/dask/dask-1.1.3.ebuild]: Please add here support for Python3.7
Should be `PYTHON_COMPAT=( python2_7 python3_{5,6,7} )`

**Comment by TheChymera** [comment]
@arthurzam solved

**Comment by juippis** [comment]
No, you broke it. Python3.7 can't work because some of the deps haven't been updated for python3.7 yet. 

Please check your contribution with repoman (it would've told about missing deps) and in future if you enable python3.7, please _test_ it before offering it to tree. I see tests are enabled for this package, too.

Now remove the python3.7 support or add its updated dependencies in this PR after testing them :P

**Comment by TheChymera** [comment]
@juippis ok, solved.

**Comment by TheChymera** [comment]
@arthurzam @juippis can you merge this?

**Comment by juippis** [comment]
We can't, sorry :(

but python project should be pretty active in here. Just a bit more patience, since it doesn't look that urgent. 

Couldn't test this because some dep didn't compile (and I don't have time to dig into it right now), but ebuild looks good to me.

**Pull request closed without merging**